src/taco-cli/cli/run.ts

```diff
@@ -167,21 +161,31 @@ class Run extends commands.TacoCommandBase {
         });
     }
 
+    private runLocal(localPlatforms?: string[]): Q.Promise<tacoUtility.ICommandTelemetryProperties> {
+        if (this.data.options["livereload"] || this.data.options["devicesync"]) {
+            // intentionally delay-requiring it since liveReload fetches whole bunch of stuff
+            return require("./liveReload").startLiveReload(!!this.data.options["livereload"], !!this.data.options["devicesync"], localPlatforms);
```

Comment:
It seems to me like we need NOT return here, since we have just set event handlers for livereload, we haven't really "ran" anything yet.